### PR TITLE
Develop java21

### PR DESCRIPTION
**Pull request by ase-101**

### CODE REVIEW

File: id-repository/id-repository-credentials-feeder/pom.xml [hunk 1]
```diff
@@ -20,8 +20,8 @@
 		<id.repository.core.version>${parent.version}</id.repository.core.version>
```
Comment (dhanendra06) [inline id-repository/id-repository-credentials-feeder/pom.xml:21]: changed the pom version 1.2.1-SNAPSHOT to 1.2.1-java21-SNAPSHOT

File: id-repository/id-repository-salt-generator/pom.xml [hunk 2]
```diff
@@ -18,8 +18,8 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```
Comment (dhanendra06) [inline id-repository/id-repository-salt-generator/pom.xml:19]: changed the pom version 1.2.1-SNAPSHOT to 1.2.1-java21-SNAPSHOT

File: id-repository/id-repository-credentials-feeder/src/main/java/io/mosip/idrepository/credentialsfeeder/config/CredentialsFeederConfig.java [hunk 3]
```diff
@@ -31,23 +40,25 @@
...
+
+	@Autowired
+	private Environment env;
 	/**
 	 * Batch config
 	 *
 	 * @return the batch configurer
 	 */
-	@Bean
+	/*@Bean
```
Comment (dhanendra06) [inline id-repository/id-repository-credentials-feeder/src/main/java/io/mosip/idrepository/credentialsfeeder/config/CredentialsFeederConfig.java:51]: remove unused code

**Review by dhanendra06 — CHANGES_REQUESTED** [review]

File: id-repository/id-repository-credentials-feeder/pom.xml (hunk 1 above)
Comment (aranaravi) [reply id-repository/id-repository-credentials-feeder/pom.xml:20]: Changed version to java21 snapshot.

File: id-repository/id-repository-salt-generator/pom.xml (hunk 2 above)
Comment (aranaravi) [reply id-repository/id-repository-salt-generator/pom.xml:18]: Changed version to java21 snapshot

File: id-repository/id-repository-credentials-feeder/src/main/java/io/mosip/idrepository/credentialsfeeder/config/CredentialsFeederConfig.java (hunk 3 above)
Comment (aranaravi) [reply id-repository/id-repository-credentials-feeder/src/main/java/io/mosip/idrepository/credentialsfeeder/config/CredentialsFeederConfig.java]: Removed the unused code.

**Review by dhanendra06 — APPROVED** [review]
Approving the pr and Junit test cases should be taken care when will merge to develop @aranaravi

**Pull request merged**